parlai/core/torch_agent.py

```diff
@@ -328,8 +363,18 @@ def add_reply(self, text):
         self._update_strings(text)
         # update history vecs
         self._update_vecs(text)
+        if self.timestep_field is not None:
```

Comment:
should this be checking `if self.timestep is not None` rather than self.timestep_field?

Reply:
since we're changing the default for `timestep_field` to `timestep`, I removed this check and make `timestep` default to the empty string instead of `None` so that the tokenizer doesn't break